_index.md

```diff
@@ -4,6 +4,6 @@ description: Resources created by engineers, for engineers.
 ---
 <!-- <div>
 <p>
-  <span class="justify-center">Section partners with university students in Computer Science related fields of study to research and write about topics that are relevant to engineers in the modern technology landscape. You can find more information and program guidelines in the <a style="text-decoration: none;" href="https://github.com/section-io/engineering-education"><span class="text-light-blue decoration-none">GitHub repository</span></a>. If you're currently enrolled in a Computer Science related field of study and are interested in participating in the program, please complete <a style="text-decoration: none;" href="https://docs.google.com/forms/d/e/1FAIpQLSfTbj3kqvEJEb5RLjqJurfbHa8ckzQx0CjRzaizblue9ZOK5A/viewform?usp=sf_link"><span class="text-light-blue decoration-none">this form</span></a>.</span>
```

Comment:
@hectorkambow this code is actually commented out. The comments start with `<!-- ` and end with `-->`. 

I think the code you're looking for lives in these two files:
[articles-header.html](https://github.com/section-engineering-education/engineering-education/blob/3fdd57a1514b446b8589d1702c1116b1478b45fc/layouts/partials/engineering-education/articles-header.html)
[eng-ed-banner.html](https://github.com/section-engineering-education/engineering-education/blob/3fdd57a1514b446b8589d1702c1116b1478b45fc/layouts/partials/engineering-education/eng-ed-banner.html)

Reply:
We can create a site variable for this EngEd message if we want to keep it consistent between the two, or keep them separate if we want them to differ. Let me know what you prefer.

Reply:
Thanks a ton Harry - I will give that a go! 

I'll be sure to ping you with updates/questions. 🚀